### PR TITLE
Challenge Complete

### PR DESCRIPTION
**Pull request by danielrileyde**
CSS - positions fixed

### CODE REVIEW

File: css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css [hunk 1]
```diff
@@ -36,29 +36,50 @@ main {
 */
 
 .box--top-left {
+  position: absolute;
+  top: 0;
```
Comment (RoyceBrouer) [inline css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css:40]: these two lines (top, bottom) are here not even necessary cause (as Jonathan has commented on mine) it is not an absolute object (I think here the box is there by default)

File: css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css [hunk 2]
```diff
@@ -36,29 +36,50 @@ main {
...
+  left: 0;
   background-color: #aed6f1;
 }
 
 .box--top-center {
+  position: absolute;
+  top: 0;
+  left: 210px;
```
Comment (RoyceBrouer) [inline css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css:48]: another possibility: left: 50%;
   transform: translate(-50%);

File: css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css [hunk 3]
```diff
@@ -36,29 +36,50 @@ main {
...
 }
 
 .box--top-right {
+  position: absolute;
+  top: 0;
+  right: 0;
   background-color: #5dade2;
 }
 
 .box--center {
```
Comment (RoyceBrouer) [inline css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css:59]: see above, other possibilty: 
left: 50%;
   top: 50%;
   transform: translate(-50%, -50%);

File: css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css [hunk 4]
```diff
@@ -36,29 +36,50 @@ main {
...
 }
 
 .box--bottom-left {
+  position: absolute;
+  bottom: 0;
+  left: 0;
   background-color: #2874a6;
 }
 
 .box--bottom-center {
```
Comment (RoyceBrouer) [inline css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css:73]: same as above

Comment (RoyceBrouer) [reply css-positioning/css-positioning_absolute-no-advanced-requirements/css/styles.css:73]: nice work see my comments but they are just alternatives, happy merging!

**Comment by danielrileyde** [comment]
Thank you @RoyceBrouer for the helpful alternatives, its good to know.

**Pull request merged**